### DISCUSSION TITLE
Pyodide Fatal Error In browser Console.

### DISCUSSION THREAD

**Discussion by gautamsarawagi** [pyodide/pyodide#4932] [Q&A]
I am working with the jupyterlite. I created a script to automatically save the file in jupyterlite and autoexecute on initial load. 

The script is working fine in all the windows but not in one of the windows particularly. Giving this error: 

![image](https://github.com/user-attachments/assets/89bad21a-9a46-4d37-9e8c-ab7265fdb88c)

![image](https://github.com/user-attachments/assets/ae1fb7a2-27e1-4cd7-9e5f-8b85ae1a7a02)

I tried to remove the pyodide from the cache. But the issue still persists. 

Would really appreciate help on this.

**Comment by ryanking13** [comment]
Without looking at the actual code, I am not sure if the error comes from Jupyterlite or Pyodide. Have you posted this to Jupyterlite issue tracker as well?

**Comment by gautamsarawagi** [comment]
I have raised the same issue in the jupyterlite forum as well. In between encountered this error.

The code for refrence: https://github.com/gautamsarawagi/file-communication-extension/blob/master/src/index.ts
